src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -350,7 +347,13 @@ public NodeSearchElementViewModel FindViewModelForNode(string nodeName)
         internal readonly DynamoViewModel dynamoViewModel;
 
         // Lucene search utility to perform indexing operations.
-        internal LuceneSearchUtility LuceneSearchUtility { get; }
+        private LuceneSearchUtility LuceneUtility
+        {
+            get
+            {
+                return LuceneSearch.LuceneUtilityNodeSearch;
```

Comment:
@RobertGlobant20 @reddyashish Fixed a regression with a one line change here, see https://github.com/DynamoDS/Dynamo/pull/14248/commits/2d2226e65333f2eed01c47a37696cc67c7ab45fe. Moving forward, we should leverage singleton instead of relying on DynamoModel. I will watch for PR check results